src/natcap/invest/cli.py

```diff
@@ -288,7 +288,7 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 
 def main(user_args=None):
-    """CLI entry point for launching InVEST runs.
+    """CLI entry point for launching InVEST runs and other useful utilities.
```

Comment:
Nice

Reply:
Some of finest docs work, right there. Haha, I didn't feel like trying to duplicate all the `parser` helptext that follows and explains all the other entry-points.